sympy/functions/elementary/tests/test_exponential.py

```diff
@@ -381,7 +381,7 @@ def test_lambertw():
     assert LambertW(2, evaluate=False).is_real
     p = Symbol('p', positive=True)
     assert LambertW(p, evaluate=False).is_real
-    assert LambertW(p - 1, evaluate=False).is_real is None
+    assert LambertW(p - 1, evaluate=False).is_real is False
```

Comment:
if we don't do this then this type of complicated case makes problem
```
>>> p = Symbol('p', positive=True)
>>> (-p + 1/S.Exp1 - 1).is_nonnegative
```
it's return ```None``` but right ans is ```False```.many other test case are possible.